documentation/deliberately_vulnerable_flask_app/true_issues.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I think we can call this `results.expected` what do you think?

Reply:
Much fancier name indeed! I am not good at naming things :P